daemon/commands.c

```diff
@@ -540,6 +540,7 @@ static void command_thread(void *arg)
 error_after_pipe_init:
     uv_close((uv_handle_t *)&async, NULL);
 error_after_async_init:
+    uv_run(loop, UV_RUN_DEFAULT);
```

Comment:
Have you put this in to force the close callback to occur before the loop close?

Reply:
Exactly.

Reply:
It makes sense in the context of this bug fix, but it will be quite difficult for A Random Person to see that when they read the code without the context. In particular, any libuv code that exists now will get used as reference when people are porting other sub-systems such as the web server. Could you add a short comment just to say why it is there?

Reply:
Done.